tests/data/dimension_associations/model_year__metric.csv

```diff
@@ -0,0 +1,11 @@
+model_year,metric
+2012,cooling
+2012,heating
+2012,water_systems
+2012,exterior_lighting
+2012,fans
+2012,heat_rejection
+2012,heating
+2012,interior_equipment
+2012,interior_lighting
+2012,pumps
```

Comment:
Why do we need associations for model_years?

Reply:
I copied everything from StandardScenarios for test purposes - not necessarily because it makes sense for this project. Maybe I should remove some of the associations.